src/Common/ChartLayouts/Standard View.xml

```diff
@@ -2,7 +2,8 @@
     <DataBrowserColumnSettings>
-      <GridColumnSettings columnName="RepositoryName" caption="Repository" visible="1" groupIndex="0" visibleIndex="0" width="0" />
+      <GridColumnSettings columnName="Category" caption="Category" visible="1" groupIndex="0" visibleIndex="0" width="0" />
+      <GridColumnSettings columnName="RepositoryName" caption="Repository" visible="1" groupIndex="-1" visibleIndex="0" width="0" />
       <GridColumnSettings columnName="Simulation" caption="Simulation" visible="0" groupIndex="-1" visibleIndex="-1" width="0" />
       <GridColumnSettings columnName="TopContainer" caption="Top Container" visible="0" groupIndex="-1" visibleIndex="-1" width="0" />
       <GridColumnSettings columnName="Container" caption="Organ" visible="1" groupIndex="1" visibleIndex="1" width="0" />
```

Comment:
please also remove the "Organ" column from Grouping

Reply:
done